src/store/mutations.js

```diff
@@ -71,6 +71,11 @@ const SET_GAS_PRICE = function(state, val) {
   store.set('gasPrice', val);
 };
 
+const SET_MINIMUM_GAS_PRICE = function(state, val) {
+  state.minimumGasPrice = val;
+  store.set('minimumGasPrice', val);
+};
+
```

Comment:
Let's try not to add a new property to the store/state. (see comment in actions.js)